### ISSUE TITLE
Status code function is broken

### ISSUE THREAD

**Issue by ricardohbin** [webpro/dyson#57]
Hi.

The mentioned way to handle status code in endpoint config doesn't works. There's some points:

- `res.send(XXX).body(YYY)` throws a `is not a function` error
- If we fix it to express way (`res.status(XXX).send(YYY)` or `res.status(XXX).end(YYY)`) it works, but raises an error in node. The famous `Error: Can't set headers after they are sent.`

As the default `render` function is just a facade `res.send()`, we can use it to responses the expected status. Example:
```
render: function(req, res) {
      if(req.params.id === "999") {
          return res.status(404).end('Feature not found');
      }
      res.send(200, res.body);
}
```

I can send a PR to fix this issue too :)

**Comment by webpro** [comment]
Fixed by @ricardohbin in #58.

**Comment by phobetron** [comment]
PR #58 broke the default functionality of `status`. If you use `status` then nothing works outside your `if` statement.
